bin/autoaggregate

```diff
@@ -77,7 +77,7 @@ def origin_for(folder):
         pattern.
     """
     base = os.path.basename(folder)
-    pattern = DEFAULT_REPO_PATTERN
+    pattern = env.get("DEFAULT_REPO_PATTERN", DEFAULT_REPO_PATTERN)
```

Comment:
Thanks for the test. I still don't like that this isn't actually setting environment variables as expected, but just using this one.

Please at least add a TODO here and a note in the README so this weird thing is clear.

Reply:
Can't this iterate in the values found in this section and set that variables in the environment?

Reply:
That would be a better solution indeed.

We usually recommend using things such as `$DEPTH_DEFAULT` or `$ODOO_VERSION`. I don't remember if the autogenerated `repos.yaml` includes those values or expands them directly.

Respecting all env variables declared would be best indeed.

Reply:
Possible updates are as follows:

1. _Use_ other environment variables (DEPTH_DEFAULT and ODOO_VERSION) on creating config: https://github.com/Tecnativa/doodba/blob/e40eb51500e3837767da440e16b3da63e3b41fb5/bin/autoaggregate#L122-L125

2. Create auto repos.yaml per environment (or per repository for simplicity) and call ``aggregate`` for each of them https://github.com/Tecnativa/doodba/blob/e40eb51500e3837767da440e16b3da63e3b41fb5/bin/autoaggregate#L138-L142

Reply:
Better option 1. We need to declare all repos in a single file to benefit from parallel downloading.

So, just expand all variables when generating the repos file.